integration-core/src/main/java/org/symphonyoss/integration/core/NullIntegration.java

```diff
@@ -37,7 +37,7 @@
  * Null pattern for integration.
  * Created by rsanchez on 21/11/16.
  */
-public class NullIntegration extends BaseIntegration implements Integration {
+public class NullIntegration extends BaseIntegration {
```

Comment:
Nice change making BaseIntegration implement Integration, the code is cleaner after this.